COMMUNITY-MANAGER.md

```diff
@@ -0,0 +1,45 @@
+# Dapr Community Managers
+
+## Charter
+
+Community Managers are responsible for managing many aspects of the Dapr community. The role is to engage and promote the Dapr community and have someone that the community can reach out to for guidance or to raise a concern. This document outlines a list of duties that a community manager can perform.
+
+### Duties
+
+A Community Manager can perform the following duties:
+
+* Managed the Discord community including posting annoucements, organizing and monitoring the Discord channels, establishing community rules, and aiding on the overall health of the Discord community. They are included in the Discord Community Manager role.
+* Review and publish Dapr blog posts submitted by the community.
+* Plan, manage and host Dapr community calls.
+* Can propose and submit changes to the Dapr website for STC approval.
+* Manage analytics tools including Google Analytics, CommonRoom.io and Scarf.sh.
+
+### Duties that require STC approval
+
+* Update the Dapr website
+* Add new analytics tools
+
+## Process to become a Community Manager
+
+Self-nominate via a [new issue in the community repository](https://github.com/dapr/community/issues/new) stating why you want to be an Community Manager, or alternatively contact an STC member. A community manager needs approval from 3 or more STC members. 
+
+## Limits and Expiration
+
+The Community Manager role expires 12 months from the date it was approved. Reapproval is then required. To minimize the number of people to coordinate between, there can a maximum of 3 active Community Managers. 
+
+## Removal
+
+The Community Manager role is permanently removed in case any of the following:
```

Comment:
I'd remove the word "permanently" as that could imply "banned for life" instead of "for now".